modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/ObjectRelationshipFormBase.tsx

```diff
@@ -29,6 +29,21 @@ import React, {useEffect, useMemo, useState} from 'react';
 
 import {defaultLanguageId} from '../../utils/constants';
 
+interface UseObjectRelationshipFormProps {
+	initialValues: Partial<ObjectRelationship>;
+	onSubmit: (relationship: ObjectRelationship) => void;
+	parameterRequired: boolean;
+}
+
+interface UseObjectRelationshipFormPros {
```

Comment:
@danvb112 Please, rename this interface to "ObjectRelationshipFormBaseProps" to keep the pattern. Remember  to move its declaration above to follow the alphabetical order. Thanks :)

Reply:
Hey @naraandrade, thanks for noticing it! clearly i just copied the name to the other one and completely forgot to change :rofl: